lbry/blockchain/database.py

```diff
@@ -16,7 +16,9 @@
 
 def make_short_url(r):
     try:
-        return f'{normalize_name(r["name"].decode())}#{r["shortestID"] or r["claimID"][::-1].hex()[0]}'
+        # fixme: we describe it as normalized but the old SDK didnt do that
```

Comment:
can you change this to `TODO`, that's what i've been using elsewhere for reminders of stuff that needs to be fixed, that way we can just grep for `TODO`

Reply:
changed, good point